app/src/components/Main/consts.js

```diff
@@ -1,6 +1,8 @@
 
 export const NAV_OPTIONS = {
+    ADMIN : "admin",
     AUCTION : "auction",
+    PUBLIC_GOODS : "public",
```

Comment:
Gotta fix formatting on render

Reply:
Resolved in abd6764eee6ad29375dff40fa1a8966aa0b4a085